src/libecalc/presentation/yaml/yaml_types/models/yaml_compressor_trains.py

```diff
@@ -125,6 +126,22 @@ class YamlSimplifiedVariableSpeedCompressorTrain(YamlCompressorTrainBase):
         title="POWER_ADJUSTMENT_CONSTANT",
     )
 
+    @model_validator(mode="after")
+    def check_control_margin(self):
+        compressor_train = self.compressor_train
+        if not isinstance(compressor_train, YamlUnknownCompressorStages):
+            for stage in compressor_train.stages:
+                if stage.control_margin:
+                    raise ValueError(
+                        f"{self.name}: {EcalcYamlKeywords.models_type_compressor_train_stage_control_margin}"
+                        f" is not allowed for {self.type}. "
+                        f"{EcalcYamlKeywords.models_type_compressor_train_stage_control_margin} is only supported for "
+                        f"{EcalcYamlKeywords.models_type_compressor_train_single_speed}, "
```

Comment:
can the list of compatible model types be retrived dynamically isntead of "hardcoding" here, in case it changes?